src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -63,7 +61,7 @@ public function showAction(Request $request, FlattenException $exception, DebugL
         $code = $exception->getStatusCode();
 
         return new Response($this->twig->render(
-            (string) $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
+            $this->findTemplate($request, $request->getRequestFormat(), $code, $showException),
```

Comment:
That's kind of useless to use a `TemplateReference` everywhere as Twig only know about plain string templates anyway. It was needed before we added support for proper Twig namespace paths, which we did earlier this year. This is a leftover that forbids to disable the templating component in 2.8/3.0.